src/components/Api.js

```diff
@@ -0,0 +1,104 @@
+export default class Api {
+  constructor(options) {
+    this._baseUrl = options.baseUrl;
+    this._headers = options.headers;
+  }
+
+  getInitialCards() {
+    //Gets the initial cards
+    return fetch(`${this._baseUrl}/cards`, {
+      headers: this._headers,
+    })
+      .then((res) => res.json())
+      .then((result) => {
+        return result;
+      })
+      .catch((err) => console.log(err));
```

Comment:
It’s better not to use `catch` in `Api` because you need to `catch` possible errors at the very end of any request and this end is in `index.js`   after all blocks `then`. And in real projects blocks `catch` usually open error popups and so on.

Please, remove it from `Api` and add it in `index.js`.